src/core_plugins/elasticsearch/lib/create_kibana_index.js

```diff
@@ -2,10 +2,10 @@ import { format } from 'util';
 import { mappings } from './kibana_index_mappings';
 
 module.exports = function (server) {
-  const client = server.plugins.elasticsearch.client;
+  const callAsKibanaUser = server.plugins.elasticsearch.getCluster('admin').callAsKibanaUser;
```

Comment:
Destructure here to make this a little more concise and less prone to typo issues:

```js
const { callAsKibanaUser } = server.plugins.elasticsearch.getCluster('admin');
```